crates/gen/src/type_namespaces.rs

```diff
@@ -1,13 +1,12 @@
-use rayon::prelude::*;
 use squote::{quote, TokenStream};
 use std::collections::*;
 
 #[derive(Default)]
 pub struct TypeNamespaces(pub BTreeMap<&'static str, crate::type_tree::TypeTree>);
 
 impl TypeNamespaces {
-    pub fn gen<'a>(&'a self) -> impl ParallelIterator<Item = TokenStream> + 'a {
-        self.0.par_iter().map(|(name, tree)| {
+    pub fn gen<'a>(&'a self) -> impl Iterator<Item = TokenStream> + 'a {
+        self.0.iter().map(|(name, tree)| {
```

Comment:
Does this not have a big impact on build time for projects that are importing a few large namespaces?

Reply:
But we could certainly parallelize those without these expensive dependencies.

Reply:
I imagine this will have negative impacts on _very_ large dependency graphs like building the entire Windows API. For my subset that I use for testing (which is about half of the total win32 API), the changes compile a bit faster (6min19sec vs 6min26sec). I think we should merge this, and then separately do the work of seeing if adding a custom parallel code gen can improve performance in the extreme cases.

Reply:
Sounds good.

Reply:
This removes quite a few dependencies. Very nice speedup. 👍🏼